examples/audio/main.py

```diff
@@ -22,17 +22,19 @@
         size_hint_y: None
         height: sp(40)
         text: 'Recording Location: ' + str(root.audio.file_path)
-
```

Comment:
why? it looks more readable if you separate widgets like that, especially if you have a lot of properties.